Doc/library/asyncio-eventloops.rst

```diff
@@ -165,7 +165,7 @@ An event loop policy must implement the following interface:
 
    .. method:: get_event_loop()
 
-      Get the event loop for the current context.
+      Get the event loop for the current coroutine.
```

Comment:
Isn't it already [there](https://github.com/python/cpython/blob/master/Doc/whatsnew/3.6.rst#L821) ?